src/zjs_webusb.c

```diff
@@ -540,7 +539,22 @@ static void check_uart(struct k_timer *timer)
     }
 }
 
-jerry_value_t zjs_webusb_init()
+static void zjs_webusb_cleanup()
+{
+    zjs_free(tx_ring);
+    zjs_free(rx_ring);
+    tx_ring = rx_ring = NULL;
+    jerry_release_value(webusb_api);
+    webusb_api = 0;
+    zjs_free(webusb_origin_url);
+    webusb_origin_url = NULL;
+}
+/*
+static const jerry_object_native_info_t webusb_module_type_info = {
+   .free_cb = zjs_webusb_cleanup
+};
+*/
```

Comment:
Why move cleanup function up here? I at least am expecting to find it at the bottom.
Also, why the commented out type_info?

Reply:
Leftover code from a previous attempt to fix this,  webusb_module_type_info was used in init and had to be first. But since its an emitter it follows a different pattern, I'll move it back and remove the un-needed code.